### PR TITLE
Added author name

### PR DESCRIPTION
**Pull request by konsim83**
Dear all, 

Many thanks for the kind offer of co-authorship of the new release. This PR contains the requested info of affiliation as requested. Please let me know about change requests.

Cheers,
Konrad

### CODE REVIEW

**Comment by bangerth** [comment]
Well deserved!

**Comment by bangerth** [comment]
Would you like to add a paragraph about your work somewhere? Maybe this could be covered in the list at the very end of section 1.

File: 9.3/paper.tex [hunk 1]
```diff
@@ -150,14 +151,19 @@
 
 \author[8]{Sebastian~Proell}
 
+\author[17]{Konrad Simon}
+  \affil[17]{Department of Mathematics /
+  Center for Earth System Research and Sustainability (CEN),
+  University of Hamburg, Grindelberg 5, 20144 Hamburg, Germany}
```
Comment (bangerth) [inline 9.3/paper.tex:157]: Please also add your email address in a separate PR.

**Comment by konsim83** [comment]
I would love to do that, the one about tge orientation that is merged I guess, right? What about the stuff that could be merged fast (another orientation thing and some work about p4est)?

**Comment by bangerth** [comment]
Too late for that, the branch has already happened a couple of weeks ago. I was thinking of the one about the orientation.

**Pull request merged**